cmd/kwil-cli/cmds/account/balance.go

```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	confirmed bool
```

Comment:
Pretty minor nit, but there's a convention I have seen in a lot of cobra repos where flags get declared within the command. So here, it would be:

```go
func balanceCmd() *cobra.Command {
    var confirmed bool
	cmd := &cobra.Command{
		Use:   "balance",
```

Reply:
Oh yeah that's a good point.  It doesn't need global scope.  Sec